lms/templates/student_account/account_settings.html

```diff
@@ -27,12 +30,17 @@
 
 <%block name="js_extra">
 <%static:require_module module_name="js/student_account/views/account_settings_factory" class_name="AccountSettingsFactory">
-    var fieldsData = ${ json.dumps(fields) };
-    var authData = ${ json.dumps(auth) };
-    var platformName = ${json.dumps(static.get_platform_name())};
+    var fieldsData = ${ fields | n, dump_js_escaped_json };
+    var authData = ${ auth | n, dump_js_escaped_json };
+    var platformName = ${ static.get_platform_name() | n, dump_js_escaped_json };
```

Comment:
Q: Is this really json?  The method and variable names suggest that it would be a string, but it was getting json-dumped.